service/routes.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func Routes(router *gin.Engine) *gin.Engine {
+	router.GET("/ping", Ping)
+	book := router.Group("/book")
+	{
+		book.PUT("", PutBook)
+		book.GET("/:id", GetBook)
+		book.POST("/:id", PostBook)
```

Comment:
The names are too coupled to the `HTTP` function
```suggestion
		book.PUT("", AddBook)
		book.GET("/:id", GetBook)
		book.POST("/:id", UpdateBook)
```